### PR TITLE
Updated the CLI DOCS

### PR DESCRIPTION
**Pull request by AlexandrouR**

### CODE REVIEW

File: docs/command_line_tools.md [hunk 1]
```diff
@@ -207,8 +209,8 @@ Generated project structure
 
 Your application code and tests will be located in the following project directories:
 
-- `./src/main/java` - Generated Java application code stub
```
Comment (cfelde) [inline docs/command_line_tools.md]: Not sure we need to remove this and the test/java line below. They are still valid when generating a Java based project, no?

Comment (AlexandrouR) [reply docs/command_line_tools.md]: yeah I will change it to kotlin

Comment (cfelde) [reply docs/command_line_tools.md]: ?

No, keep the old + the new you already added

**Comment by cfelde** [comment]
LGTM

**Pull request merged**